cli/viam/main.go

```diff
@@ -57,7 +58,7 @@ func main() {
 			&cli.StringFlag{
 				Name:    "config",
 				Aliases: []string{"c"},
-				Usage:   "Load configuration from `FILE`",
+				Usage:   "load configuration from `FILE`",
```

Comment:
Opting for lowercase usage texts where possible.

Reply:
I couldn't find where this was defined in a quick search, but if doing this here, please also lowercase this sentence:

```
bananajr:~ andf$ viam --help | grep "or help"
   help, h        Shows a list of commands or help for one command
```

Reply:
I wanted to but that message is from the underlying `urfave/cli` library, so I can't change it.